test/Google.Api.Gax.Grpc.Tests/RetryTest.cs

```diff
@@ -65,27 +60,34 @@ public class RetryTest
             await scheduler.RunAsync(async () =>
             {
                 var callSettings = CallSettings.FromCallTiming(CallTiming.FromRetry(retrySettings));
-                var retryingCallable = callable.WithRetry(scheduler.Clock, scheduler);
-                var result = await Call(async, retryingCallable, new SimpleRequest { Name = name }, callSettings);
+                SimpleResponse result;
+                if (serverStreaming)
+                {
+                    var retryingCallable = server.ServerStreamingCallable.WithRetry(scheduler.Clock, scheduler);
+                    result = await Call(async, retryingCallable, new SimpleRequest { Name = name }, callSettings);
```

Comment:
Presumably the different types involved here mean we can't factor out the common code?

Reply:
They are indeed different types; but I'll see about doing a small refactor.